### ISSUE TITLE
get() fails always withAfterEffectsError

### ISSUE THREAD

**Issue by eyalw** [BenGaumond/after-effects#13]
Replicate:

```
> node
> ae = require('ae');
> ae.execute(() => get())
.then(val => console.log(val))
.catch(err => console.log(err);
```
Outputs...
```
{ [AfterEffectsError: types, if supplied, are expected to be non-anonymous Function definitions.] name: 'AfterEffectsError' }
```

**Comment by eyalw** [comment]
Looks like the error originated from here (get.js:31). I don't understand this code, it referes to an undefined var 'type' (I validated this, it is undefined) - and uses it for a condition in a loop - where the condition will give the same result on all iterations + it will throw the error on the first loop.
@BenGaumond - Any idea?

![image](https://cloud.githubusercontent.com/assets/608669/18286759/e2f540a4-747c-11e6-8ed0-11deaf347a05.png)

**Comment by eyalw** [comment]
It looks like the intention was to do: 
![image](https://cloud.githubusercontent.com/assets/608669/18287595/6b6a3824-7480-11e6-8d2a-3456d61191a3.png)

**Comment by BenGaumond** [comment]
I can't believe this has been wrong for so long! Thank you very much for pointing this out, this will be fixed in 0.4.11

**Comment by BenGaumond** [comment]
0.4.11 is up.
